firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionGenerator.kt

```diff
@@ -32,43 +32,39 @@ internal data class SessionInfo(
 
 /**
  * The [SessionGenerator] is responsible for generating the Session ID, and keeping the
- * [SessionInfo] up to date with the latest values.
+ * [SessionState] up to date with the latest values.
  *
  * @hide
  */
-internal class SessionGenerator(collectEvents: Boolean) {
+internal class SessionGenerator(private var collectEvents: Boolean) {
```

Comment:
wow! We can call parameters with access protectors? `private var` is so cool! TIL!